### ISSUE TITLE
LLVM 7.0 is updated

### ISSUE THREAD

**Issue by loquat** [wqweto/VBLLVM#1]
will you update this project?

**Comment by loquat** [comment]
it seems 8.0 is current version, maybe can update via SVN

**Comment by wqweto** [comment]
Yes, this is planned and I've been waiting for the release on llvm-dev mailing list.

There are particularly some bugs in the linker tear-down that made using it in-process impossible (from IDEs incl. VB6 IDE)

Will have to find some time this weekend to merge upstream.
